Orange/tests/test_variable.py

```diff
@@ -99,16 +121,23 @@ def test_adjust_decimals(self):
         a.val_from_str_add("5.1234")
         self.assertEqual(a.str_val(4.654321), "4.6543")
 
+    def dont_pickle_anonymous_variables(self):
```

Comment:
def dont_pickle_anonymous_variables(self)

As is does not start with test_, it is not considered a test method. And there is already

VariableTest. test_dont_pickle_anonymous_variables

which does that.